lms/static/sass/shared-v2/_header.scss

```diff
@@ -125,3 +125,27 @@
         }
     }
 }
+
+.doc-link {
+    position: relative;
```

Comment:
I would remove all the styles from this except for the float right and the margin. There's nothing special about the doc link that means that it should override all the standard patterns for links IMO. In general, we want pattern library pages to not need additional styling.